packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -23,17 +23,28 @@ export interface MeetingCallControlBarProps {
   onPeopleButtonClicked: () => void;
   mobileView: boolean;
   disableButtonsForLobbyPage: boolean;
+  meetingCallControlOptions?: MeetingCallControlOptions;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
-  // Disable the default participants button as meetings composite has its own participants button.
+  // Set the desired control buttons from the meetings composite. particiapantsButton is always false since there is the peopleButton.
   let callControlsOptions: CallControlOptions | false = {
     participantsButton: false,
-    screenShareButton: props.mobileView ? false : { disabled: props.disableButtonsForLobbyPage }
+    screenShareButton:
+      props.mobileView || !props.meetingCallControlOptions?.screenShareButton
+        ? false
+        : { disabled: props.disableButtonsForLobbyPage },
+    displayType: props.meetingCallControlOptions?.displayType,
+    microphoneButton: !props.meetingCallControlOptions?.microphoneButton ? false : true,
+    cameraButton: !props.meetingCallControlOptions?.cameraButton ? false : true,
+    devicesButton: !props.meetingCallControlOptions?.devicesButton ? false : true
   };
+  // Set flags for the chat button and people button specific to the meeting control bar.
+  const chatButton = props.meetingCallControlOptions?.chatButton === false ? false : true;
```

Comment:
I'm guessing this is trying to check if (1) the prop was explicitly set, (2) it was set to `false`.

That can simply be `!(props.meetingCallControlOptions?.chatButton === false)`

* I'd  inline that check in line 71. Separating that logic from its use isn't helpful.
* I'd also create a helper function below to make the intent clearer.

  const isEnabled(option: boolean | undefined): boolean => !(option === false); 


Then, line 71 becomes

  { isEnabled(props.meetingCallControlOptions?.chatButton) && (<ChatButton>....</ChatButton>) }

Reply:
done! thanks for the suggestion! 😊